cmd/credentialUtil.go

```diff
@@ -646,6 +648,29 @@ func getCredentialType(ctx context.Context, raw rawFromToInfo, cpkOptions common
 // ==============================================================================================
 // pipeline factory methods
 // ==============================================================================================
+func createClientOptions(logLevel pipeline.LogLevel) azcore.ClientOptions {
+	logOptions := ste.LogOptions{}
+	if azcopyScanningLogger != nil {
+		logOptions.LogOptions = pipeline.LogOptions{
+			Log:       azcopyScanningLogger.Log,
+			ShouldLog: func(level pipeline.LogLevel) bool { return level <= logLevel },
```

Comment:
IIRC track-2 sdk does not have this level based logging. We need to register what events we want and then create a callback method that receives those events. How are we taking care of info/error logging in that case based on level we have configured.

Reply:
We are using the AzCopy logger here and choosing to register on none of the Track 2 log events. imo the AzCopy logger is too different from the Track 2 SDK default logger to be able to take advantage of it. We can revisit this after all the APIs are migrated. It will be a fairly simple fix.